cmd/modern/root/config/add-user.go

```diff
@@ -170,3 +171,23 @@ func (c *AddUser) run() {
 	uniqueUserName := config.AddUser(user)
 	output.Info(localizer.Sprintf("User '%v' added", uniqueUserName))
 }
+
+func (c *AddUser) LogTelemtry() {
+	eventName := "config-add-user"
+	properties := map[string]string{}
+	if c.username != "" {
+		properties["username"] = "set"
```

Comment:
is it easier to build telemetry reports if a property always has a value instead of omitting it when not used? What do the report queries look like?